src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml

```diff
@@ -16,9 +16,20 @@
                 <span asp-validation-for="Role.Name" class="text-danger"></span>
             </div>
 
+            @if (Model.UsersInRole > 0)
+            {
+                <a asp-action="@nameof(RolesController.UsersInRole)"
+                   asp-route-id="@Model.Role.Id"
+                   class="btn btn-default">See users in this role (@Model.UsersInRole)</a>
+            }
+            else
+            {
+                <button class="btn btn-default" disabled>No users in this role</button>
+            }
+
             <div class="row">
                 <div class="col-xs-6">
-                    <h3>Available Permissions</h3>
+                    <h3>Available Permissions (@Model.UnselectedPermissions?.Count())</h3>
```

Comment:
This should be an `h2` with the `class="h3"`.  With those changes the view looks the same. This is just an accessibility issue we should get in routine of.